econsa/tests/test_quantile_measures.py

```diff
@@ -0,0 +1,122 @@
+"""Test for quantile based global sensitivity measures.
+
+Analytical values of linear model with normally distributed variables
+are used as benchmarks for verification of numerical estimates.
+"""
+import numpy as np
+import pytest
+from numpy.testing import assert_almost_equal
+from scipy.stats import norm
+
+from econsa.quantile_measures import mcs_quantile
+
+
+@pytest.fixture
+def first_example_fixture():
+    """First example test case."""
+
+    def func1(args):
```

Comment:
Glad to see you getting started with tests, please use (or add if neeed) all test functions to  `temfpy`. @loikein  can help in the process.

Reply:
Ok!